components/MultipleChoice.py

```diff
@@ -1,20 +1,34 @@
 import kivy
 import json
 import random
-from Question import Question
+from kivy.uix.screenmanager import Screen
+from kivymd.app import MDApp
+from components.card import Card, CardButton
+from components.Question import Question
 
 
 class MultipleChoice(Question):
     def __init__(self, question_text, question_id, question_audio, explanation_text, explanation_audio,
-                 image_options, correct_answer, selected_choices):
-        Question.__init__(self, question_text, question_id, question_audio, explanation_text, explanation_audio)
+                 image_options, correct_answer):
+        Question.__init__(self, question_text, question_id,
+                          question_audio, explanation_text, explanation_audio)
         self.image_options = image_options
         self.correct_answer = correct_answer
-        assert [i for i in self.correct_answer not in self.image_options] == []
-        self.selected_choices = selected_choices
+        # assert [i for i in self.correct_answer not in self.image_options] == []
```

Comment:
@michaelzhang0608 I left a comment in your previous PR for what this did. Can you explain? It crashes the program